velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -261,6 +261,10 @@ void Writer::close() {
   arrowContext_->stagingChunks.clear();
 }
 
+void Writer::abort() {
+  VELOX_NYI("abort function for Parquet writer is not supported yet");
```

Comment:
Do not leave it throwing, just implement it properly

Reply:
I got the comment from @xiaoxmeng to put `VELOX_NYI` instead of implementing, I changed accordingly because I think we can leave it to someone who's an expert in Parquet to do it properly

Reply:
Where are we going to call `abort()`?  If it will make their code to throw in a normal run, we should implement it for them, maybe with a `LOG(WARNING)` to let me look at it.  Silently failing their code is not nice.

Reply:
No, it's not in a normal run right now, this is a new API and they would start to use it later on

Reply:
Actually as soon as we start using it in `HiveDataSink` you will need to implement it.

Reply:
CC: @majetideepak @JkSelf